src/Surfnet/StepupRa/RaBundle/Tests/Security/Authentication/Token/SamlTokenTest.php

```diff
@@ -49,4 +52,81 @@ public function saml_token_is_correctly_serialized_and_unserialized()
 
         $this->assertEquals($samlToken, $deserialized);
     }
+
+    /**
+     * @test
+     * @group authorization
+     * @group sraa
```

Comment:
These `@group`s are new, is this a conscious decision? Perhaps the existing `@group security` is enough.